calc.py

```
@@ -62,8 +62,13 @@ def opPrecedence(self,op1,op2):
 print("hello")
 
 
+print("hihi" , 10//0)
+
 prnt("hello")
```

Ruff (0.8.2)

Undefined name `prnt`

(F821)